### DISCUSSION TITLE
Best practice for continuous updates

### DISCUSSION THREAD

**Discussion by btrepp** [argoproj/argo-cd#8879] [Q&A] [answered]
Hi,

I have an Argo cd setup, and ci that is building a static website in a docker containers.

Just wondering, is there a way to have Argo always be 'the latest image'. Eg if I publish something new, it will deploy?.

At the moment I think I would need to tag the image, and use update the Argo application to the new tag, this is great for controls/approvals, but for lower risk items, it's a bit of extra work. Especially if my aim is 'nightly'/latest/unstable.

Is there any best practice for this use case?.

Thanks :)

**Comment by alexmt** [comment]
I think you are looking for https://github.com/argoproj-labs/argocd-image-updater . The image updater automates updating manifests in git/application CRD everytime when new image is pushed to image registry.

**Reply by btrepp** [reply]
@alexmt thanks alex, that was a good lead but I ended up going a different way.

**Comment by btrepp** [comment, marked as answer]
Best practice seems to be 'don't' do this in K8S.

As I learnt more, it really seems the more gitops approach, would be have the versions in git, and use automation to update that file in your repo, and get it pushed. Then let gitops take over.

So for anyone else, write scripts and automation to update your git, that way everything is deterministic and deploys the same
